walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,63 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void optionalAttendeeNotAvail() {
```

Comment:
I am in favor of naming tests to include the expected outcome, this allows anyone looking at the code to understand fully what the test should do. This is a debated topic, see https://stackoverflow.com/questions/155436/unit-test-naming-best-practices for examples.
No need to change your test methods here, but sometimes it can make tests easier to follow

Reply:
Thanks for the resource! I actually was unsure of whether there were any test naming conventions so this clears things up.